cpp/src/slice2php/Main.cpp

```diff
@@ -262,7 +267,8 @@ CodeVisitor::visitClassDefStart(const ClassDefPtr& p)
     }
 
     // Emit the type information.
-    _out << nl << type << " = IcePHP_defineClass('" << scoped << "', '" << escapeName(abs) << "', " << p->compactId()
+    const string abs = p->mappedScoped("\\\\");
```

Comment:
We can use raw literal:

```suggestion
    const string abs = p->mappedScoped(R"(\\)");
```

Reply:
I thought about it for all these backslashes, but they didn't make it any shorter, so I just left them.
I'm fine adding it here, where it at least doesn't make it any longer.